website_docs/quick_start.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Ruby Quick Start Guide"
+weight: 24
+description: >
+  <img width="35" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Ruby_SDK.svg"></img>
+  A language-specific implementation of OpenTelemetry in Ruby.
+---
+
+[OpenTelemetry for Ruby](https://github.com/open-telemetry/opentelemetry-ruby) can be used to add automatic and manual instrumentation to your applications.
+Automatic instrumentation is enabled by adding [instrumentation packages](https://github.com/open-telemetry/opentelemetry-ruby#instrumentation-libraries).
+Manual instrumentation can be added using the OpenTelemetry API.
+
+## Quick Start
+
+### Requirements
+
+These instructions will explain how to set up automatic and manual instrumentation for a Ruby service.
+In order to follow along, you will need:
+
+- Ruby 2.5 or higher
+- Docker Compose
+
+### Installation
+
+The first step add these gems to your Gemfile:
+
+```ruby
+gem 'opentelemetry-sdk'
+gem 'opentelemetry-exporter-otlp'
+gem 'opentelemetry-instrumentation-all'
+```
+
+The inclusion of `opentelemetry-instrumentation-all` in the above list provides instrumentations for several frameworks such as Rails and Sinatra as well as database drivers and HTTP [libraries](https://github.com/open-telemetry/opentelemetry-ruby#instrumentation-libraries).
+
+### Initialization
+
+The OpenTelemetry initialization needs to happen early in your application lifecycle.
+For Rails applications, the usual way to initialize OpenTelemetry is in a Rails initializer.
+For other Ruby services, perform this initialization as early as possible in the start-up process.
+
+OpenTelemetry initialization:
+
+```ruby
+# config/initializers/opentelemetry.rb
+require 'opentelemetry/sdk'
+require 'opentelemetry/exporter/otlp'
+require 'opentelemetry/instrumentation/all'
+OpenTelemetry::SDK.configure do |c|
+  c.service_name = '<YOUR_SERVICE_NAME>'
+  c.use_all() # enables all instrumentation!
+end
+```
+
+Now that you have setup your application to perform tracing you'll need to configure the SDK to export the traces to somewhere that will vizualize them for you.
+
+### Exporting Traces
+
+The following section assumes you are new to OpenTelemetry or do not currently use a vendor that supports distributed tracing using OTLP. Please refer to your vendor's product documentation if you would like to export your traces to a vendor for analysis and vizualization.
+
+For the purposes of this tutorial you will configure an OpenTelemetry collector that will receive the traces and vizualize them using Jaeger or Zipkin UI.
+
+First, start up an example system:
+
+```bash
+$> git clone git@github.com:open-telemetry/opentelemetry-ruby.git; \
+    cd open-telemetry-ruby/examples/otel-collector; \
+       docker-compose up -d
```

Comment:
![MAGIC](https://user-images.githubusercontent.com/517302/132950565-07f4b182-c4d4-4909-a97a-45362025e3af.gif)

I'll test this out this weekend.

Reply:
You will have to switch to this branch in order for that to work.